src/utils.rs

```diff
@@ -19,12 +19,24 @@ pub static DELETE_REGEX: Lazy<Regex> = Lazy::new(|| {
         .expect("failed to compile regex")
 });
 
-pub fn format_message(links: impl IntoIterator<Item = (Entry, Flavor)>) -> String {
+pub fn format_message(
+    links: impl IntoIterator<Item = (Entry, Flavor)>,
+    user: Option<&User>,
+) -> String {
     links
         .into_iter()
-        .map(|(entry, flavor)| {
+        .enumerate()
+        .map(|(index, (entry, flavor))| {
+            let prefix = match index {
+                0 => match user {
+                    Some(user) => format!("{} found help", html::user_mention_or_link(user)),
+                    None => "Found help".to_string(),
+                },
+                _ => "... and".to_string(),
```

Comment:
Would look weird in multi-reply
```
@username found help for X in Y docs:
<link>

... and for Z in Y docs:
<link>

... and for W in Q docs:
<link>
```

Maybe use this one?
```
{flavor} docs for {topic}:
{link}

{flavor} docs for {topic}:
{link}

{flavor} docs for {topic}:
{link}

[summoned by {user}]
```
`Custom` should be capitalized in this case.